### PR TITLE
create, load data and handle event tab my music

### PR DESCRIPTION
**Pull request by kpt96**
create, load data and handle event tab my music 
https://edu-redmine.framgia.vn/issues/166

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -11,4 +11,4 @@ org.gradle.jvmargs=-Xmx1536m
 # This option should only be used with decoupled projects. More details, visit
 # http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
 # org.gradle.parallel=true
-API_KEY=INSERT_YOUR_API_KEY_HERE
+API_KEY=a7Ucuq0KY8Ksn8WzBG6wj4x6pcId6BpU
```
Comment (DoanVanToan) [inline gradle.properties]: keep old code.

File: app/src/main/java/com/framgia/soundcloud/screen/mymusic/LoadMyMusicContract.java [hunk 2]
```diff
@@ -0,0 +1,15 @@
+package com.framgia.soundcloud.screen.mymusic;
+
+import com.framgia.soundcloud.data.model.Track;
+
+import java.util.List;
+
+public interface LoadMyMusicContract {
```
Comment (DoanVanToan) [inline app/src/main/java/com/framgia/soundcloud/screen/mymusic/LoadMyMusicContract.java]: rename to LocalMusicContract

File: app/src/main/java/com/framgia/soundcloud/screen/mymusic/LoadMyMusicContract.java [hunk 3]
```diff
@@ -0,0 +1,15 @@
+package com.framgia.soundcloud.screen.mymusic;
```
Comment (DoanVanToan) [inline app/src/main/java/com/framgia/soundcloud/screen/mymusic/LoadMyMusicContract.java]: renam to localmusic

File: app/src/main/java/com/framgia/soundcloud/screen/mymusic/LoadMyMusicContract.java [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+import java.util.List;
+
+public interface LoadMyMusicContract {
+    interface Presenter {
+        void loadMyMusic();
+    }
+
+    interface View {
+        void showMyMusic(List<Track> tracks);
```
Comment (DoanVanToan) [inline app/src/main/java/com/framgia/soundcloud/screen/mymusic/LoadMyMusicContract.java]: rename to showMusic

File: app/src/main/java/com/framgia/soundcloud/screen/mymusic/LoadMyMusicContract.java [hunk 5]
```diff
@@ -0,0 +1,15 @@
+package com.framgia.soundcloud.screen.mymusic;
+
+import com.framgia.soundcloud.data.model.Track;
+
+import java.util.List;
+
+public interface LoadMyMusicContract {
+    interface Presenter {
+        void loadMyMusic();
```
Comment (DoanVanToan) [inline app/src/main/java/com/framgia/soundcloud/screen/mymusic/LoadMyMusicContract.java]: rename to loadMusic

File: app/src/main/java/com/framgia/soundcloud/ItemMyMusicClickListener.java [hunk 6]
```diff
@@ -0,0 +1,5 @@
+package com.framgia.soundcloud;
+
+public interface ItemMyMusicClickListener {
```
Comment (DoanVanToan) [inline app/src/main/java/com/framgia/soundcloud/ItemMyMusicClickListener.java]: interface này em nên để là nested class của adapter sẽ dễ đọc dễ tìm kiếm hơn

**Comment by DoanVanToan** [comment]
commented

**Comment by kpt96** [comment]
ready

**Comment by kpt96** [comment]
ready

**Pull request merged**